packages/node_modules/@ciscospark/widget-meet/src/components/call-active/index.js

```diff
@@ -32,16 +32,19 @@ export default function ActiveCall({
 
   const buttons = [
     {
+      accessibilityLabel: `mute audio`,
```

Comment:
Yea, based on the link you posted from MDN, i'd say let's capitalize the accessibility labels. I was just suggesting that we keep that consistent.